tools/get-change-log-for-release.js

```diff
@@ -11,9 +11,11 @@ const main = async () => {
     validateCommandLineArguments(params);
 
     const gitLogs = await getGitLogs(params.from, params.to);
-
-    const outputContent = generateOutputContent(gitLogs, params.to);
-
+    const outputContent =
+        params.kind === 'csv'
+            ? generateOutputContent(gitLogs, params.to)
+            : generateTextContent(gitLogs);
+    console.log(outputContent);
```

Comment:
Did we want to leave this console.log here? Totally fine if it makes sense during generation output, just wanted to make sure :)

Reply:
I find it useful to validate that it did what I wanted without having to open the file but could see it getting cumbersome if  there are dozens of commits. I could go either way.